docs/source/private_data_tutorial.rst

```diff
@@ -473,39 +594,103 @@ command to commit the chaincode definition. This command will also deploy the
 collection definition to the channel.
 
 We are ready to use the chaincode after the chaincode definition has been
-committed to the channel. Because the marbles private data chaincode contains an
-initiation function, we need to use the `peer chaincode invoke <commands/peerchaincode.html?%20chaincode%20instantiate#peer-chaincode-instantiate>`__ command
-to invoke ``Init()`` before we can use other functions in the chaincode.
+committed to the channel. 
 
 :guilabel:`Try it yourself`
 
-1. Run the following commands to commit the definition of the marbles private
+1. Run the following commands to commit the definition of the asset transfer private
 data chaincode to the channel ``mychannel``.
 
 .. code:: bash
 
     export ORDERER_CA=${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
     export ORG1_CA=${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
     export ORG2_CA=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
-    peer lifecycle chaincode commit -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name marblesp --version 1.0 --sequence 1 --collections-config ../chaincode/marbles02_private/collections_config.json --signature-policy "OR('Org1MSP.member','Org2MSP.member')" --tls --cafile $ORDERER_CA --peerAddresses localhost:7051 --tlsRootCertFiles $ORG1_CA --peerAddresses localhost:9051 --tlsRootCertFiles $ORG2_CA
+    peer lifecycle chaincode commit -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name private --version 1.0 --sequence 1 --collections-config ../asset-transfer-private-data/chaincode-go/collections_config.json --signature-policy "OR('Org1MSP.member','Org2MSP.member')" --tls --cafile $ORDERER_CA --peerAddresses localhost:7051 --tlsRootCertFiles $ORG1_CA --peerAddresses localhost:9051 --tlsRootCertFiles $ORG2_CA
 
 
 When the commit transaction completes successfully you should see something
 similar to:
 
 .. code:: bash
 
-    2020-01-06 16:24:46.104 EST [chaincodeCmd] ClientWait -> INFO 001 txid [4a0d0f5da43eb64f7cbfd72ea8a8df18c328fb250cb346077d91166d86d62d46] committed with status (VALID) at localhost:9051
-    2020-01-06 16:24:46.184 EST [chaincodeCmd] ClientWait -> INFO 002 txid [4a0d0f5da43eb64f7cbfd72ea8a8df18c328fb250cb346077d91166d86d62d46] committed with status (VALID) at localhost:7051
+    2020-09-12 06:40:35.126 CDT [chaincodeCmd] ClientWait -> INFO 001 txid [701468d079f8deded428a67f84e9bda6f6dcc894b006ee27a4d47b1dafdf5621] committed with status (VALID) at localhost:7051
+    2020-09-12 06:40:35.162 CDT [chaincodeCmd] ClientWait -> INFO 002 txid [701468d079f8deded428a67f84e9bda6f6dcc894b006ee27a4d47b1dafdf5621] committed with status (VALID) at localhost:9051
+
+.. _pd-register-identities:
+
+Register identities
```

Comment:
This section is another reason to have the transaction flow at the top of the tutorial. It comes out of left field otherwise.

Reply:
Agree. Done.